saleor/settings.py

```diff
@@ -620,8 +620,8 @@ def get_host():
     RAZORPAY: {
         "module": "saleor.payment.gateways.razorpay",
         "config": {
-            "store_card": False,
-            "auto_capture": None,
+            "store_card": get_bool_from_env("RAZORAY_STORE_CARD", False),
```

Comment:
I'm not sure about this. We have no way of telling through the API or any other way the credit card information.

Reply:
Could you please elaborate?

Reply:
There razorpay payment provider is JS based and will not give us any information about the payment apart from the payment method, e.g. "card"

Reply:
We only need _customer_id_ and I found sth like [this](https://razorpay.com/docs/payment-methods/cards/saved-cards/).
It seems that we might reuse source from razorpay from my very quick research.